### DISCUSSION TITLE
Setup Connection to NGX server - not working :/

### DISCUSSION THREAD

**Discussion by SuperUschi** [LeoKlaus/Paperparrot#136] [Q&A]
Hi community,

I have issues to connect to my local ngx server and get a "server responded: 400" and "URLSession returned error: <private>".
Since there is no wiki or setup document I am a bit lost ;-) 
I have used IP:8010 or IP:8080 with right credentials. I have full access to my server via webpage.
Maybe I am missing some settings at docker (portainer +compose)?

Many thanks for support

**Comment by LeoKlaus** [comment]
Hi SuperUschi,

I've moved this to an issue (#137 ), so I'll close this discussion for now.
